src/main/webapp/app/entities/required-document/required-document-detail.tsx

```diff
@@ -39,6 +39,18 @@ export class RequiredDocumentDetail extends React.Component<IRequiredDocumentDet
               <Translate contentKey="serviceNetApp.requiredDocument.srvc">Srvc</Translate>
             </dt>
             <dd>{requiredDocumentEntity.srvcName ? requiredDocumentEntity.srvcName : ''}</dd>
+            <dt>
+              <span id="externalDbId">
+                <Translate contentKey="serviceNetApp.requiredDocumentEntity.externalDbId" />
```

Comment:
are you sure there should be Entity at the end of translate message?
See other occurences